.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
On reflection, I'm a bit confused by this. This will get people to put the content of the proposed CIP _in the PR description_, rather than in a file in the PR. Is that what you want?

Reply:
Hmmm. Good point.

Reply:
Moved to a separate file, mentioned in the body of CIP-0001